lib/Elastica/Transport/Guzzle.php

```diff
@@ -93,6 +94,7 @@ public function exec(Request $request, array $params)
             [
                 'request_header' => $request->getMethod(),
                 'http_code' => $res->getStatusCode(),
+                'body' => $responseBody,
```

Comment:
I'm looking at this line and wondering if it could have any other side effects. Thoughts?

Reply:
@ruflin unlikely but your heart is in the right place: nothing apart from tests was accessing the array passed to 'setTransportInfo'